Kickstarter-iOS/Features/Discovery/Controller/DiscoveryNavigationHeaderViewController.swift

```diff
@@ -211,11 +216,6 @@ internal final class DiscoveryNavigationHeaderViewController: UIViewController {
       |> UILabel.lens.backgroundColor .~ .ksr_support_100
       |> UILabel.lens.isAccessibilityElement .~ false
       |> UILabel.lens.textColor .~ discoveryPrimaryColor()
-      |> UILabel.lens.font %~~ { _, label in
```

Comment:
Was this delete intentional? (Just because above it says `initial style`, unclear if it's also supposed to be able to mutate in `bindViewModel`).

Reply:
This is github being unhelpful - it's deleted from `bindStyles` and is still in `bindViewModel`. So the font will still be updated when the view model is updated; it just won't be reset if the traitcollection changes and `bindStyles` gets called again.